Rewriting {{wombatUrl}} when serving data (#18)
SnowdogApps/wombat: README.md

{
  "defaultLang": "en", // Fallback language when request is send without `lang` query param.
  "allowedOrigins": [], // List of domains used to set Access-Control-Allow-Origin (CORS) response header

>   "wombatUrl": "http://google.com", // String to substitute all occurences of {{wombatUrl}} in *.json and *.md files.

Reviewer: To nie dotyczy tylko json/md, mimo ze tylko takie poki co wspieramy. Mysle tez ze warto uzywac jezyka tozsamego z tym co uzywamy w kodzie, wiec `replace` jest lepszym wyborem niz `substitute`. Warto tez podac jakis bardziej realny przyklad URLa.

```suggestion
  "wombatUrl": "http://api.your.app", // String to replace all occurences of {{wombatUrl}} with given URL.
```
